docs/reference/advanced/metrics-in-spans.md

```diff
@@ -0,0 +1,209 @@
+# Aggregating Metrics in Spans
+
+Logfire lets you aggregate counter and histogram metrics within the current active span and its ancestors. This is particularly useful for calculating totals for things like LLM token usage or costs on higher-level operations, making it easier and more efficient to query this data without processing individual child spans.
+
+This guide will walk you through how to enable this feature and use it with both custom metrics and automated instrumentation.
+
+!!! note
+    This is an experimental feature. The API and data format may change in future releases. We welcome your feedback to help us prioritize improvements and stability.
+
+    This only works for metrics recorded within the same process. It does not aggregate metrics across distributed traces.
+
+## Enabling Metric Aggregation
+
+<!-- TODO add pydantic-ai version -->
```

Comment:
After next release (since https://github.com/pydantic/pydantic-ai/pull/3013 is merged)